notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -926,7 +920,8 @@
         "id": "jbRf1WoH_vbY"
       },
       "source": [
-        "You can use the `aiplatform.get_pipeline_df()` method to access run metadata. Here, you'll get metadata for the last two runs of the same pipeline and load it into a Pandas DataFrame. The `mlmd-pipeline` parameter here refers to the name you gave your pipeline in the pipeline definition:"
+        "Use `aiplatform.get_pipeline_df()` method to retrieve the metadata for the last two runs of the pipeline. And then, load it into a Pandas DataFrame. \n",
```

Comment:
remove "And". (note: don't start a sentence with "And")

Reply:
Noted